tools/pipelines/register-es-pipelines.js

```diff
@@ -93,7 +93,7 @@ if(!isService && ctx.contact != null && ctx.contact.length > 0 && ctx.contact[0]
   ok++
 }
 // this checks for multilingual Organizations (GN 4.2.3+)
-if(!isService && ctx.contact != null && ctx.contact.length > 0 && ctx.contact[0].organisationObject != null && ctx.contact[0].organisationObject.default != '') {
+if(!isService && ctx.contact != null && ctx.contact.length > 0 && ctx.contact[0].organisationObject != null && ctx.contact[0].organisationObject.default != null && ctx.contact[0].organisationObject.default != '') {
```

Comment:
Thx for the fix, working great now for `dataset/a3774ef6-809d-4dd1-984f-9254f49cbd0a`!